packages/progress-monitor/src/browser/progress-protocol.ts

```diff
@@ -0,0 +1,24 @@
+/********************************************************************************
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+export interface ProgressReport {
```

Comment:
Are this and packages/progress-monitor/src/browser/progress-monitor-impl.ts different? Or was this leftover from before? Or used in different ways? (Just wondering because the interface is the same name)

Reply:
Fixed